### PR TITLE
ARM support

### PR DESCRIPTION
**Pull request by alvarocabanas**
- Add ARM support

### CODE REVIEW

File: build/.goreleaser.yml
Comment (roobre) [inline build/.goreleaser.yml:45]: Seems to collide with https://github.com/newrelic/nri-jmx/pull/79, I'd wait until that's merged

**Review by varas — CHANGES_REQUESTED** [review]
https://github.com/newrelic/nri-jmx/pull/80/files#r649331274 dep should be >= 1.5.3

File: build/.goreleaser.yml
Comment (alvarocabanas) [reply build/.goreleaser.yml:45]: Yes, I am aware, I will wait until that is merged to rebase and get the >= 1.5.3

Comment (alvarocabanas) [reply build/.goreleaser.yml]: I have rebased this branch and now it has the changed done by Cristian.

**Review by cristianciutea — APPROVED** [review]

**Pull request merged**
